QRCoder/Base64QRCode.cs

```diff
@@ -43,25 +33,81 @@ public string GetGraphic(int pixelsPerModule, string darkColorHtmlHex, string li
 
         public string GetGraphic(int pixelsPerModule, Color darkColor, Color lightColor, bool drawQuietZones = true, ImageType imgType = ImageType.Png)
         {
-            var base64 = string.Empty;
-            using (Bitmap bmp = qr.GetGraphic(pixelsPerModule, darkColor, lightColor, drawQuietZones))
+            if (imgType == ImageType.Png)
             {
-                base64 = BitmapToBase64(bmp, imgType);
+                var pngCoder = new PngByteQRCode(QrCodeData);
+
+                byte[] pngData;
+                if (darkColor == Color.Black && lightColor == Color.White)
+                {
+                    pngData = pngCoder.GetGraphic(pixelsPerModule, drawQuietZones);
+                }
+                else
+                {
+                    byte[] darkColorBytes;
+                    byte[] lightColorBytes;
+                    if (darkColor.A != 255 || lightColor.A != 255)
```

Comment:
The PNG file created by the pre-existing code is written differently if there is an alpha channel.  It is likely we could simplify the approach if desired.

Reply:
You're right - by using rgba anyway we would probably bloat pngs that would otherwise created with RGB only. Then let's keep your implementation. It's fine for me. :-)